arm_controllers/TrajectoryPhase[] phases
